docs/tutorials/_tabs/setting-up-ethernet-over-bluetooth-on-ubuntu-14.04.md

```diff
@@ -30,3 +29,16 @@ If you are using an older version, please upgrade.
     depend on the Bluetooth adapter on your host computer.
 
     {% include screenshot.html source="/images/brickman/bluetooth-confirm-passkey.png" %}
+
+---
+
+These instructions use blueman. Install it with sudo apt-get install blueman. Blueman should work parallel to Network Manager, too. I assume you have already paired your PC with your EV3 brick using the discreption above.
+
+Using blueman it is quite easy to establish an ethernet connection with internet.
+
+1.  Start blueman (if not already started)
+2.  You should see a bluetooth icon in the task bar. Left click on it and choose "local services"
```

Comment:
It sounds like you are not using the default Unity desktop. In Unity, left- or right-click does not matter on the notification area icons. Also "Local Services" is not an option.

Since most Ubuntu users are probably using Unity, I would like these instructions to reflect that.